test/DataRepo-test.js

```diff
@@ -0,0 +1,173 @@
+import { expect } from 'chai';
+import DataRepo from '../src/DataRepo';
+import { tripsSampleData , tripsForOneUserSample } from './Sample-trips-data'
+import Destination from '../src/Destination';
+import destinationSampleData from './Sample-destination-data';
+import travelerSampleData from './Sample-traveler-data'
+import dayjs from "dayjs"
+
+describe ('DataRepo', () => {
+
+    let dataRepo;
```

Comment:
you can also format with commas instead of saying let each time  like this for less lines: 
`let tripsForOneUser, travelersSample, currentUser, pastTripsForSampleUser` 
.... etc